src/assets/sprites/fing.svg

```diff
@@ -1,5 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="80" height="80" viewBox="0 0 80 80">
   <g fill="none" fill-rule="evenodd">
+    <circle fill="#1fa8f1" cx="40" cy="40" r="40"></circle>
```

Comment:
Why do you add a colored circle in the svg when you already have a wrapper with border-radius and background-color in CSS?